### DISCUSSION TITLE
Cannot specify template repository after misspecifying non template repository

### DISCUSSION THREAD

**Discussion by skojaku** [community/community#67661] [GitHub Education]
### Hi everyone,

I created a new classroom and the first assignment. I misspecified the starter github repository to a non-template and this results in an error in creating the assignment. 

Now,  I tried to fix it by redoing everything, including recreating the classroom. However,  I'm stacked with the first window which asks name of assignment, and visibility and cannot go to the next page, where I can change the template repo to fix the error.

If I clicked "Next" button, a top banner pops up saying 

```
Starter code repository is not a template repository.
```

Could this be a bug or is there any way to reset the template repository?

**Comment by samus-aran** [comment]
Hi @skojaku 👋 

A quick suggestion: Are you able to press "Cancel" or clear your browser cache and try again?

If it still isn't working are you able to share the name of your classroom repo & assignment for the internal team to investigate 👍

**Reply by skojaku** [reply]
Thanks! Clearing the cache works!!!!
